src/frontend/components/ProductsListing/ProductsListing.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useProducts } from "../../contexts";
+import { ProductCard } from "../ProductCard/ProductCard";
+import "./ProductsListing.css";
+
+function ProductsListing() {
+  const { products } = useProducts();
+  return (
+    <div class="heading-products-container">
+      {/* Heading */}
+      <h1>All Products</h1>
+
+      {/* <!-- All Product Cards --> */}
+      <div class="products-container">
+        {products.map(
+          ({ _id, name, price, categoryName, imageUrl, rating }) => (
+            <ProductCard
+              key={_id}
+              _id={_id}
```

Comment:
dont really need to pass the id to the component if thats not gonna be used

Reply:
Yeah, you're right.

Reply:
@MarkVed17 You might need the `id` later.